addons/arsenal/stringtable.xml

```diff
@@ -689,6 +691,7 @@
         </Key>
         <Key ID="STR_ACE_Arsenal_buttonImportTooltip">
             <English>Import current / default loadouts</English>
+            <German>Jetzige / StandarD Ausrüstungen importieren.</German>
```

Comment:
"Standard"

Reply:
"Ausrüstungs-sets importieren" ?
Is that the normal "Loadouts" button or the "Import" button? Or is that for the BI Arsenal -> ACE Arsenal conversion?